components/security.rst

```diff
@@ -36,6 +36,10 @@ used separately:
 ``symfony/security-csrf``
     It provides protection against `CSRF attacks`_.
 
+``symfony/security-guard``
+    It brings many layers of authentication together, making it much easier
+    to create complex authentication systems.
```

Comment:
Thank you for your contribution.

I like your paragraph but would like to get rid of the “easier” part.
What do you think of sth like this:
It brings many layers of authentication together, to create complex authentication systems.

Reply:
Thank you for the review.
I removed the 'easier' part and rephrased.